src/renderer/components/swap/Swap.tsx

```diff
@@ -501,11 +499,8 @@ export const Swap = ({
   // depends on users balances of source asset
   // Decimal always <= 1e8 based
   const maxAmountToSwapMax1e8: BaseAmount = useMemo(() => {
-    // make sure not logged in user can play around with swap
-    if (unlockedWallet) return assetToBase(assetAmount(Number.MAX_SAFE_INTEGER, sourceAssetAmountMax1e8.decimal))
-
-    return sourceAssetAmountMax1e8
-  }, [unlockedWallet, sourceAssetAmountMax1e8])
+    return Utils.maxAmountToSwapMax1e8(sourceAssetAmountMax1e8, swapFees)
```

Comment:
Don't remove the case to handle locked / notimported wallet users - they still can play around.